package.json

```diff
@@ -37,10 +37,15 @@
     "valid-url": "^1.0.9"
   },
   "devDependencies": {
+    "@financial-times/athloi": "github:ysugimoto/athloi",
```

Comment:
Temporary use my fork. need change to actual version after https://github.com/Financial-Times/athloi/pull/58 has been merged and released.